sympy/solvers/tests/test_solvers.py

```diff
@@ -2290,3 +2290,16 @@ def test_issue_19509():
         -d + a + sqrt(-b + c),
         d + a - sqrt(-b - c),
         d + a + sqrt(-b - c)]
+
+
+def test_issue_4886():
+    Ra , Rb = var('Ra , Rb',positive = True)
```

Comment:
Do you need to use `pow`? Why not `Ra = R**2  + a**2`

`var` should not be used in library code. Use `symbols` instead.

Follow PEP8 conventions throughout, a few pointers -
There should be a space after `,`, and no space before, there should be spaces around `+` and `-`, assignment operator should not have extra leading spaces to make it aligned, only one space before and one after is fine, assigning values to function parameters should not have any spaces around assignment operator (`positive=True`)